sandbox-backend/src/main/resources/db/migration/V0000026__addEmails_Subscriptions.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE employee_category(
```

Comment:
It is category of notifications for selected employee but not category of employees (admins, users, etc). So I think that it is better use something like `employee_notification_category`

Reply:
done